src/PeckaCodingStandard/ruleset.xml

```diff
@@ -45,6 +45,8 @@
 	<rule ref="Generic.PHP.ForbiddenFunctions"/>
 	<rule ref="Generic.PHP.UpperCaseConstant"/>
 
+	<rule ref="../../../../escapestudios/symfony2-coding-standard/Symfony2/Sniffs/Formatting/BlankLineBeforeReturnSniff.php"/>
```

Comment:
Naimportuj si standard pomocí https://github.com/peckadesign/CodingStandard/blob/master/src/PeckaCodingStandard/ruleset.xml#L4 a potom to můžeš použít bez těch cest. Když to tak není, není možné používat nastavení těch pravidel, viz předchozí má oprava https://github.com/peckadesign/CodingStandard/commit/b06660791d25a7d61dee26afcd83c0eb660335d6

Reply:
`<rule ref="Symfony.Formatting.BlankLineBeforeReturn"/>`

Reply:
`	<config name="installed_paths" value="../../slevomat/coding-standard,../../escapestudios/symfony2-coding-standard"/>`

Reply:
jj, už jsem na to taky přišel, vycházel jsem ze starší verze